_includes/v22.2/known-limitations/drop-owned-by-limitations.md

```diff
@@ -0,0 +1,7 @@
+- If the [role](security-reference/authorization.html#roles) for which you are trying to `DROP OWNED BY` was granted a privilege using the [`GRANT SYSTEM ...`](grant.html#grant-global-privileges-on-the-entire-cluster) statement, the error shown below will be signalled.  The workaround is to use [`SHOW SYSTEM GRANTS FOR {role}`](show-system-grants.html) and then use [`REVOKE SYSTEM ...`](revoke.html#revoke-global-privileges-on-the-entire-cluster) for each privilege in the result. For more information about this known limitation, see [cockroachdb/cockroach#88149](https://github.com/cockroachdb/cockroach/issues/88149).
+
+    ~~~
+    ERROR: cannot perform drop owned by if role has synthetic privileges; foo has entries in system.privileges
+    SQLSTATE: 0A000
+    HINT: perform REVOKE SYSTEM ... for the relevant privileges foo has in system.privileges
+    ~~~
```

Comment:
@rmloveland I've re-used this in the 22.2 Known Limitations. Could you please consider adding the following to this PR or creating follow-up issues:

1. Update the include to contain both Known Limitations currently listed on the KL page.
2. Consider that the linked section in `grant.html` refers only to "global privileges" but the Known Limitation and public issue refers to "synthetic privileges"—can we reconcile these in the docs somehow? Not sure if they are synonyms and we should choose one, etc. (I am happy to create a Security issue for this if you prefer.)

Reply:
1. yes
2. sounds good, I can reconcile

Reply:
Following up on this:
1. This is done in the latest changes I just pushed.
2. After some research I think reconciling "system" vs. "synthetic" is too big to do in this PR.  I've filed https://cockroachlabs.atlassian.net/browse/DOC-6325 with a bunch of details about work I think we need to do to document the "synthetic" vs. "system" thing properly.  If nothing else at least that convo should happen there, I think so there's more explicitness and clarity about our use of this terminology going forward. (They appear to be synonyms now but we need to be clear they will continue to be going forward, and if we decide to keep "synthetic" there's add'l work around explaining "synthetic" and also then "descriptors", etc etc. See issue)

Reply:
Gonna resolve this convo but come yell at me on DOC-6325